### ISSUE TITLE
The optimizer doesn't seem so good

### ISSUE THREAD

**Issue by Huster-Hq** [milesial/Pytorch-UNet#337]
Ask the author if you think there is improvement in the optimizer in this code. I think the learning rate drops too fast！

**Comment by milesial** [comment]
I did not have much compute available to try many hyperparameters. You're free to change the optimizer and the learning rate scheduler to fit your dataset.

**Comment by Huster-Hq** [comment]
Ok, thank you very much
